docs/getting-started.md

```
@@ -0,0 +1,83 @@
+t = np.linspace(0, 1, fs)
+# Composite signal: 100Hz + 1000Hz
+signal = np.sin(2 * np.pi * 100 * t) + np.sin(2 * np.pi * 1000 * t)
+
+# Apply 1/3 octave filter bank
+spl, freq = octavefilter(signal, fs=fs, fraction=3)
+
+print(f"Bands: {freq}")
+print(f"SPL [dB]: {spl}")
+```
+
+<picture><source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/jmrplens/PyOctaveBand/main/.github/images/signal_response_fraction_3_dark.png"><img src="https://raw.githubusercontent.com/jmrplens/PyOctaveBand/main/.github/images/signal_response_fraction_3.png" width="80%"></picture>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)